### PR TITLE
install: add -D example

### PR DESCRIPTION
**Pull request by Krutonium**
- [ ] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/install.md [hunk 1]
```diff
@@ -23,3 +23,7 @@
 - Copy files and apply access/modification times of source to the destination:
 
 `install --preserve-timestamps {{path/to/source}} {{path/to/destination}}`
+
+- Copy files and create the directories in the path if they don't exist
```
Comment (CleanMachine1) [inline pages/common/install.md:27]:
```suggestion
- Copy files and create the directories in the path if they don't exist:
```

File: pages/common/install.md [hunk 2]
```diff
@@ -23,3 +23,7 @@
 - Copy files and apply access/modification times of source to the destination:
 
 `install --preserve-timestamps {{path/to/source}} {{path/to/destination}}`
+
+- Copy files and create the directories in the path if they don't exist:
```
Comment (CleanMachine1) [inline pages/common/install.md:27]:
```suggestion
- Copy files and create the directories in the destination directory if they don't exist:
```

Comment (navarroaxel) [reply pages/common/install.md:27]: It's not clear... too many `directories`

Comment (navarroaxel) [reply pages/common/install.md:27]:
```suggestion
- Copy files and create the directories in the destination path if they don't exist:
```

**Review by navarroaxel — APPROVED** [review]

**Comment by navarroaxel** [comment]
@CleanMachine1 could you?

File: pages/common/install.md [hunk 3]
```diff
@@ -23,3 +23,7 @@
 - Copy files and apply access/modification times of source to the destination:
 
 `install --preserve-timestamps {{path/to/source}} {{path/to/destination}}`
+
+- Copy files and create the directories at the destination if they don't exist:
+
+`install -D {{path/to/source}} {{path/to/destination}}`
```
Comment (EmilySeville7cfg) [inline pages/common/install.md:29]:
```suggestion
`install -D {{path/to/source_file1 path/to/source_file2 ...}} {{path/to/destination_directory}}`
```

Please fix any other code samples to be consistent with this one.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

File: pages/common/install.md (hunk 3 above)
Comment (Krutonium) [reply pages/common/install.md]: Done

**Review by EmilySeville7cfg — APPROVED** [review]

**Comment by EmilySeville7cfg** [comment]
Merged, thanks! 🚀

**Pull request merged**